File: JavaScript/Fase3.js
const ciutat1 = "Barcelona";
const ciutat2 = "Madrid";
const ciutat3 = "Valencia";
const ciutat4 = "Malaga";
const ciutat5 = "Cadis";
const ciutat6 = "Santander";

const arrayCiutats = [];
const arrayCiutatsModificades = [];

arrayCiutats.push(ciutat1, ciutat2, ciutat3, ciutat4, ciutat5, ciutat6);

arrayCiutats.forEach(ciutat => {
    ciutat = ciutat.replace(/a/g, '4');
    arrayCiutatsModificades.push(ciutat);
});

console.log(arrayCiutatsModificades.sort());

File: Java/src/Fase3.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.Collections;
import java.util.List;

public class Fase3 {
	public static void main(String[] args) {

		String ciutat1 = "Barcelona";
		String ciutat2 = "Madrid";
		String ciutat3 = "Valencia";
		String ciutat4 = "Malaga";
		String ciutat5 = "Cadis";
		String ciutat6 = "Santander";
		
		List<String> arrayCiutats            = new ArrayList<>();
		List<String> arrayCiutatsModificades = new ArrayList<>();

		arrayCiutats.addAll(Arrays.asList(ciutat1, ciutat2, ciutat3, ciutat4, ciutat5, ciutat6));
		
		for(String ciutat : arrayCiutats) {
			arrayCiutatsModificades.add(ciutat.replaceAll("a", "4"));
		}
		
		Collections.sort(arrayCiutatsModificades);
		System.out.print(arrayCiutatsModificades);
	}
}
